libsel4/include/interfaces/sel4.xml

```diff
@@ -48,6 +48,35 @@
                 description="Number of slots into the node at which capabilities start being placed."/>
             <param dir="in" name="num_objects" type="seL4_Word"
                 description="Number of capabilities to create."/>
+            <error name="seL4_DeleteFirst" description="A capability exists in the destination window of the CNode."/>
+            <error name="seL4_FailedLookup">
+                <description>
+                    The <texttt text="root,"/> <texttt text="node_index,"/> or <texttt text="node_depth"/> is invalid <docref>(see <autoref label="s:cspace-addressing"/>).</docref>
+                </description>
+            </error>
+            <error name="seL4_IllegalOperation">
+                <description>
+                    The <texttt text="_service"/> is a CPtr to a capability of the wrong type.
+                </description>
+            </error>
+            <error name="seL4_InvalidArgument">
+                <description>
+                    The <texttt text="size_bits"/> is too big or too small for the requested object type.
```

Comment:
This one generates a `seL4_RangeError`. https://github.com/seL4/seL4/blob/5fd9619fb589e672fd968e57ceb8a10f548dc98a/src/object/untyped.c#L80-L86

Reply:
Ah, but it can also generate a `seL4_InvalidArgument`, right. So it should stay here. Sorry for the noise.